src/file_writer.rs

```diff
@@ -0,0 +1,63 @@
+use std::fs::File;
+use std::io::{self, Write};
+use std::path::PathBuf;
+
+use crate::common::LineEntry;
+
+// I think that we should create a backup copy
+pub fn write(path: &PathBuf, lines: Vec<LineEntry>) -> io::Result<()> {
+    let mut file = File::create(path)?;
+
+    for line in lines[..lines.len() - 1].iter() {
+        writeln!(file, "{}", line.raw_string)?;
+    }
+
+    Ok(())
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::common::FileEntry;
+
+    use std::env::temp_dir;
+    use std::fs::{self, remove_file};
+
+    #[test]
+    fn write_test() {
+        let file_name = String::from(".env");
+        let path = temp_dir().join(&file_name);
+
+        let fe = FileEntry {
+            path: path.clone(),
+            file_name,
+            total_lines: 3,
+        };
+
+        let lines = vec![
+            LineEntry {
+                number: 1,
+                file: fe.clone(),
+                raw_string: String::from("A=B"),
+            },
+            LineEntry {
+                number: 2,
+                file: fe.clone(),
+                raw_string: String::from("Z=Y"),
+            },
+            LineEntry {
+                number: 3,
+                file: fe,
+                raw_string: String::from("\n"),
+            },
+        ];
+
+        assert!(write(&path, lines).is_ok());
+        assert_eq!(
+            b"A=B\nZ=Y\n",
+            fs::read(path.to_str().unwrap()).unwrap().as_slice()
```

Comment:
We don't use the `unwrap` method.

Reply:
It is test, but OK.

Reply:
I replaced `unwrap` (here and in one more place).